queries.sql

```diff
@@ -56,4 +56,58 @@ SELECT name,escape_attempts FROM animals WHERE weight_kg>10.5;
      select species,min(weight_kg) from animals group by species;
      select species,max(weight_kg) from animals group by species;
      /* the average number of escape attempts per animal type of those born between 1990 and 2000 */
-      select species,avg(escape_attempts) from animals where date_of_birth between '1990-01-01' and '2000-01-01'  group by species;
\ No newline at end of file
+      select species,avg(escape_attempts) from animals where date_of_birth between '1990-01-01' and '2000-01-01'  group by species;
+
+    /* Melody Pond's */
+      select name,full_name
+      from animals
+      join owners
+      on animals.owner_id = owners.id where owners.full_name='Melody Pond';
+      
+      /* pokemon list */
+       select animals.name as animal , species.name as type
+       from animals
+       join species
+       on animals.species_id = species.id where species.name = 'Pokemon';
+
+    /* number of animals of per species */
+      select species.name, count(animals.id)
+      from animals
+      join species
+      on animals.species_id = species.id group by species.name;
+    /* owner and animal */
+      select owners.full_name, animals.name
+      from animals
+      join owners
+      on animals.owner_id = owners.id;
```

Comment:
- [ ] Your query is running, but there's a potential issue. Currently, you're querying to get animals per owners, but there could be a scenario where someone like `Jodie Whittaker` doesn't have any animals. As per the project requirement, you should "List all owners and their animals, remember to include those that don't own any animal." This means your query should also display owners who don't have any animals. Please update your code to fulfill this requirement, ensuring that all owners are included in the results, whether they have animals or not. Keep up the good work! 🐾👥🦔